### DISCUSSION TITLE
Can't run Flameshot on Tiling Window Manager LeftWM

### DISCUSSION THREAD

**Discussion by SunnyOd** [flameshot-org/flameshot#2466] [Q&A]
Hi,

I can see this should work as lots of TWMs are using it, but it's not working for me on LeftWM. 

I'm running Manjaro Gnome with Leftwm installed over it. Have all latest updates

This is the error I'm getting:

`~ >>> flameshot gui

QApplication: invalid style override 'kvantum' passed, ignoring it.
	Available styles: HighContrastInverse, HighContrast, Adwaita-HighContrastInverse, Adwaita-HighContrast, Adwaita-Dark, Adwaita, bb10dark, bb10bright, cleanlooks, gtk2, cde, motif, plastique, qt5ct-style, Windows, Fusion
~ >>>`

Searched here and online for the error but not found anything related to flameshot. Any ideas?

Thanks!
Sunny

**Comment by asdrubalivan** [comment]
Hello, I'm getting the same error right now on Manjaro after an update
